src/huggingface_hub/cli/lfs.py

```diff
@@ -33,58 +34,39 @@
 logger = logging.get_logger(__name__)
 
 
-class LfsCommands(BaseHuggingfaceCLICommand):
-    """
-    Implementation of a custom transfer agent for the transfer type "multipart"
-    for git-lfs. This lets users upload large files >5GB 🔥. Spec for LFS custom
-    transfer agent is:
-    https://github.com/git-lfs/git-lfs/blob/master/docs/custom-transfers.md
-
-    This introduces two commands to the CLI:
-
-    1. $ hf lfs-enable-largefiles
-
-    This should be executed once for each model repo that contains a model file
-    >5GB. It's documented in the error message you get if you just try to git
-    push a 5GB file without having enabled it before.
-
-    2. $ hf lfs-multipart-upload
-
-    This command is called by lfs directly and is not meant to be called by the
-    user.
-    """
-
-    @staticmethod
-    def register_subcommand(parser: _SubParsersAction):
-        enable_parser = parser.add_parser("lfs-enable-largefiles", add_help=False)
-        enable_parser.add_argument("path", type=str, help="Local path to repository you want to configure.")
-        enable_parser.set_defaults(func=lambda args: LfsEnableCommand(args))
-
-        # Command will get called by git-lfs, do not call it directly.
-        upload_parser = parser.add_parser(LFS_MULTIPART_UPLOAD_COMMAND, add_help=False)
-        upload_parser.set_defaults(func=lambda args: LfsUploadCommand(args))
+"""
+Implementation of a custom transfer agent for the transfer type "multipart" for git-lfs.
 
+Commands:
+- hf lfs-enable-largefiles
+- hf lfs-multipart-upload (internal; called by git-lfs)
+"""
 
-class LfsEnableCommand:
-    def __init__(self, args):
-        self.args = args
 
-    def run(self):
-        local_path = os.path.abspath(self.args.path)
-        if not os.path.isdir(local_path):
-            print("This does not look like a valid git repo.")
-            exit(1)
-        subprocess.run(
-            "git config lfs.customtransfer.multipart.path hf".split(),
-            check=True,
-            cwd=local_path,
-        )
-        subprocess.run(
-            f"git config lfs.customtransfer.multipart.args {LFS_MULTIPART_UPLOAD_COMMAND}".split(),
-            check=True,
-            cwd=local_path,
-        )
-        print("Local repo set up for largefiles")
+def lfs_enable_largefiles(
+    path: Annotated[
+        str,
+        typer.Argument(
+            ...,
+            help="Local path to repository you want to configure.",
+        ),
+    ],
```

Comment:
```suggestion
    path: Annotated[
        str,
        typer.Argument(help="Local path to repository you want to configure.")
    ],
```

adding `...` as default value is not needed I think (since `...` is already the default value for arguments). I feel that it's more concise to read without it and understandable 


(not the only occurrence like this one, would be good to do a pass to remove them all)